.vscode/settings.json

```diff
@@ -17,9 +17,10 @@
     "logs/**": true
   },
   "python.testing.pytestArgs": [
-    "tests"
+    "."
   ],
   "python.testing.unittestEnabled": false,
   "python.testing.pytestEnabled": true,
-  "quarto.render.renderOnSave": true
+  "quarto.render.renderOnSave": true,
+  "python.testing.cwd": "${worskpaceFolder}/benchmarks/"
```

Comment:
I think I might prefer that tests for benchmarks be run separately from the main tests (as we've already got a fairly long running test suite which holds up every PR, etc.). I think the idea should be that you can run tests for a benchmark locally but they aren't run automatically when testing the package?

Reply:
I think thats a mistaken addition!